packages/spark-core/src/spark-core.js

```diff
@@ -91,11 +91,11 @@ const SparkCore = AmpState.extend({
   },
 
   authenticate(...args) {
-    return this.credentials.authenticate(...args);
+    return this.credentials.initiateLogin(...args);
```

Comment:
the credentials plugin in spark-core doesn't have initiateLogin, which is why deprecated autheticate instead of removing it.